src/tables.c

```diff
@@ -2225,3 +2233,192 @@ const byte gammatable[5][256] =
+    11,-10,-9,15,-4,0,-6,9,-16,12,-2,7,5,-6,5,6,-3,11,14,8,-8,-2,-6,12,
+    -11,-13,8,-13,-12,11,-7,-4,-10,6,12,9,-5,4,3,-8,4,5,-3,11,15,10,-6,1,
+    -2,-14,-5,-5,-15,-2,0,-7,9,14,10,-4,4,3,-8,3,5,-3,11,15,10,-4,3,1,
+    -10,1,2,-7,7,12,6,-8,-1,-3,-14,-3,-2,-10,5,10,5,-9,-1,-2,-12,0,2,-5,
+    11,-15,14,2,12,12,4,-15,-10,-15,3,12,11,1,14,-15,11,-4,4,2,-9,3,6,-1,
+    -16,-9,-11,10,-11,-8,-15,2,9,7,-3,9,12,6,-9,-1,-2,-12,2,6,1,-13,-3,-3,
+    -11,4,10,7,-5,7,10,4,-11,-3,-3,-12,2,8,5,-7,5,8,2,-12,-3,-3,-11,5,
+    12,10,0,14,-13,-16,4,-16,-13,14,1,11,13,7,-8,1,2,-6,11,-13,-13,11,-6,2,
+    1,-8,8,15,14,5,-12,-5,-6,-15,0,8,7,-2,14,-10,-10,14,-2,7,8,1,-14,-4,
+    -2,-8,11,-10,-7,-11,10,-9,-4,-6,-16,-1,7,7,-1,-15,-5,-3,-7,13,-6,-1,-3,-11,
+    5,13,15,10,-3,9,15,13,4,-12,-2,0,-4,-16,-2,5,5,-2,-16,-4,1,-1,-10,7,
+    -15,-11,-14,8,-8,1,4,0,-10,6,14,-15,13,3,-13,-4,-1,-5,-14,2,12,-16,14,6,
+    -9,3,9,8,2,-11,3,11,13,9,-1,15,-6,-1,-2,-9,11,-7,2,5,2,-6,12,-7,
+    0,2,-2,-11,7,-13,-6,-4,-8,15,1,14,-10,-8,-10,14,1,-16,-7,-3,-4,-9,12,-3,
+    9,-16,-14,-16,9,-3,12,-9,-3,-1,-4,-12,8,-8,3,10,12,10,3,-7,10,-9,-1,4,
+    4,0,-8,12,-4,7,15,-13,-13,15,7,-4,12,-7,2,7,9,7,1,-8,11,-5,7,15,
+    -12,-10,-12,15,7,-5,12,-7,4,11,15,-16,14,9,1,-10,8,-9,3,12,-14,-11,-10,-12,
+    15,7,-3,-16,1,15,-6,3,10,14,15,14,11,6,-2,-12,7,-7,8,-10,1,10,-15,-10,
+    -7,-6,-7,-10,-14,11,3,-7,14,0,-15,1,15,-5,6,15,-9,-3,2,6,8,9,9,7,
+    5,1,-4,-10,15,7,-2,-13,8,-4,-16,2,-12,4,-12,4,-14,1,14,-5,7,-13,-1,10,
+    -12,-2,8,-14,-5,4,13,-10,-1,7,-16,-8,1,10,-14,-5,4,13,-10,0,10,-12,-2,9,
+    -11,0,13,-7,7,-11,3,-13,3,-12,5,-9,11,-1,-12,9,0,-8,-15,11,6,3,0,-1,
+    -1,0,3,7,12,-13,-4,6,-15,-2,13,-3,15,3,-8,15,8,3,0,-1,0,3,7,14,
+    -9,3,-16,0,-14,6,-4,-11,-15,14,15,-15,-9,-1,9,-10,6,-7,14,7,2,0,1,4,
+    11,-11,2,-14,5,-4,-11,-14,-14,-10,-4,6,-12,5,-6,-14,14,13,-16,-9,1,15,1,-9,
+    -15,14,-16,-11,-1,13,-2,-12,14,13,15,-10,1,-15,5,-3,-6,-5,1,11,-5,14,7,4,
+    6,13,-8,9,-2,-7,-8,-3,6,-11,9,2,0,3,12,-6,14,7,5,9,-14,1,-10,-16,
+    -16,-10,1,-14,9,6,9,-14,1,-11,-16,-15,-8,6,-7,-13,-13,-7,6,-7,-13,-13,-6,7,
+    -5,-11,-10,-2,13,2,-2,1,11,-4,-12,-13,-7,6,-5,-10,-7,3,-12,12,12,-12,2,-7,
+    -9,-4,9,-2,-6,-2,10,-3,-7,-4,7,-5,-10,-7,4,-8,-13,-9,3,-9,-12,-7,6,-4,
+    -6,0,-16,7,8,-15,3,-3,0,12,1,-1,6,-10,14,-16,-5,15,12,-13,2,-5,-3,9,
+    -2,-4,4,-10,-15,-11,3,-5,-3,8,-2,-3,5,-8,-11,-4,12,7,12,-5,-11,-8,6,-2,
+    0,13,4,6,-14,9,11,-10,13,14,-5,-15,-13,0,-9,-7,7,-1,3,-15,11,-16,0,-5,
+    1,-13,-15,-7,14,13,-7,15,-14,0,-7,-1,-16,13,-10,11,12,-8,-15,-10,6,3,13,2,
+    3,-15,12,-14,5,4,-16,9,14,-1,-3,8,0,4,-10,-12,-1,-10,-5,13,12,-8,-15,-9,
+    10,11,-7,-12,-3,-13,-10,7,6,-14,11,-14,7,10,-6,-7,5,-2,6,-4,0,-14,-15,-1,
+    -5,6,-2,5,-6,-2,-16,-16,-1
+};
+
+void ChangeTrigTablesToDoom1_0()
+{
+	int i, j, count;
+
+    for (i = 0; i < 66; i += 2)
+        finesine1_2[finesineto1_0[i]] += finesineto1_0[i+1];
+
+    for(i = 0, j = 0, count = 1; count; j += count)
```

Comment:
Some of the formatting here is sloppy. Please try to match the style described in `HACKING` and please also don't introduce new tab characters.

Reply:
Done in https://github.com/chocolate-doom/chocolate-doom/pull/1207/commits/d8f6bcf685da5f096480d21cc0162de06d1b6f24 .